grumpy-runtime-src/testing/str_test.py

```diff
@@ -18,7 +18,7 @@
 
 # Test Add
 assert "foo" + "bar" == "foobar"
-assert "foo" + u"bar" == u"foobar"
+assert "foo" + "bar" == "foobar"
```

Comment:
This removal is unnecessary in all currently supported versions of CPython (2.7, 3.4, 3.5, 3.6, 3.7)

Reply:
The meaning changed. Should be `b"foo" + u"bar" == u"foobar"` I guess.